mail_outplacement_report/models/outplacement.py

```diff
@@ -1,35 +1,89 @@
-from odoo import models, api, _, fields
 import logging
+from odoo import models, api, fields, _
+from datetime import datetime, timedelta, date
 
 _logger = logging.getLogger(__name__)
 
+LOCAL_TZ = 'Europe/Stockholm'
+
+
 class HrEmployee(models.Model):
     _inherit = "hr.employee"
 
     def action_send_email(self):
         self.ensure_one()
         ir_model_data = self.env['ir.model.data']
-        _logger.warn("DAER: ir_model_data: %s" % ir_model_data)
         try:
-            
             template_id = ir_model_data.get_object_reference('mail_outplacement_report', 'email_template_login_employee')[1]
-            _logger.warn("DAER: template_id1: %s" % template_id)
         except ValueError:
             template_id = False
-            _logger.warn("DAER: template_id: %s" % template_id)
-        template_browse =self.env['mail.template'].browse(template_id).send_mail(self.id, force_send=True)
-        _logger.warn("DAER: template_browse: %s" % template_browse)
+        template_browse = self.env['mail.template'].browse(template_id).send_mail(self.id, force_send=True)
+
 
 class Outplacement(models.Model):
     _inherit = "outplacement"
 
+    date = fields.Date(string="Meeting date", help="Date of meeting")
+    time = fields.Float(string="Meeting time", default=' " "', help="t.ex. 15:30")
```

Comment:
`default=' " " '` This cant be converted to a float.
I think that this row is the causing the error when installing the module, 
https://github.com/vertelab/odoo-mail/pull/11#issuecomment-770707327